.github/workflows/release.yml

```diff
@@ -47,6 +47,13 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+      
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL

```

Comment:
If we can easily wrap our existing build process with this, can we also add it to the PR workflow. We only run this at release time, but it would be much better to get CodeQL warnings or errors at PR time.